client/blocks/login/two-factor-authentication/verification-code-form.scss

```diff
@@ -1,11 +1,24 @@
 .two-factor-authentication__small-print {
 	clear: both;
-	font-size: $font-body-extra-small;
-	padding: 1em 0;
+	padding: 2em 0 0 0;
+	text-wrap: pretty;
```

Comment:
Let's not add this here, we should add this only once, globally, through #100475. If there's a typographic widow in the mean time, so be it.

Reply:
Thanks for the reminder. It is fixed in 8e0ca84795d876dcd8f42ac64e15c236ed7d4a97